src/plugins/controls/public/actions/index.ts

```diff
@@ -6,6 +6,6 @@
  * Side Public License, v 1.
  */
 
-export const ACTION_EDIT_CONTROL = 'editLegacyEmbeddableControl';
+export const ACTION_EDIT_CONTROL = 'editDataControl';
```

Comment:
`editLegacyEmbeddableControl` isn't necessary anymore, as the legacy edit action has been deleted and replaced with the React control edit action